manifests/j/JeffreyPfau/mGBA/0.9.3/JeffreyPfau.mGBA.installer.yaml

```diff
@@ -0,0 +1,23 @@
+# Created with YamlCreate.ps1 v2.0.4 $debug=QUSU.7-2-0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: JeffreyPfau.mGBA
+PackageVersion: 0.9.3
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+InstallerType: inno
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/mgba-emu/mgba/releases/download/0.9.3/mGBA-0.9.3-win32-installer.exe
+  InstallerSha256: 935FD31DDDBE7D0D535254DED0AB00580D24E16623558F9BCEA65D45ABE66AFE
+- Architecture: x86
```

Comment:
```suggestion
- Architecture: x64
```

Reply:
If you use wingetcreate you will better avoid these kinds of errors:

`wingetcreate update JeffreyPfau.mGBA -u https://github.com/mgba-emu/mgba/releases/download/0.9.3/mGBA-0.9.3-win32-installer.exe https://github.com/mgba-emu/mgba/releases/download/0.9.3/mGBA-0.9.3-win64-installer.exe`